pylint/checkers/base/basic_checker.py

```diff
@@ -680,11 +680,7 @@ def visit_call(self, node: nodes.Call) -> None:
     @utils.only_required_for_messages("assert-on-tuple", "assert-on-string-literal")
     def visit_assert(self, node: nodes.Assert) -> None:
         """Check whether assert is used on a tuple or string literal."""
-        if (
-            node.fail is None
-            and isinstance(node.test, nodes.Tuple)
-            and len(node.test.elts) == 2
-        ):
+        if isinstance(node.test, nodes.Tuple) and len(node.test.elts) > 0:
             self.add_message("assert-on-tuple", node=node)
```

Comment:
```suggestion
            self.add_message("assert-on-tuple", node=node, confidence=HIGH)
```
This will require an import and regenerating the functional tests. (We update the content may as well upgrade the confidence)